backend/src/main/resources/liquibase/db-changelog-1.0.xml

```diff
@@ -180,6 +180,23 @@
         </addColumn>
     </changeSet>
 
+    <changeSet id="vmem-config-blob-mapping-table" author="anandJ">
+        <preConditions onFail="MARK_RAN">
+            <not>
+                <tableExists tableName="versioning_modeldb_entity_mapping_config_blob"/>
+            </not>
+        </preConditions>
+        <createTable tableName="versioning_modeldb_entity_mapping_config_blob">
```

Comment:
lets ensure this runs on OSS setup as well.

Reply:
working on OSS